File: backend/temp.js
const nodemailer = require("nodemailer");

// Create a transporter object using Gmail
const transporter = nodemailer.createTransport({
    service: "gmail",
    auth: {
        user: "deepakjgenai@gmail.com", // Your email
        pass: "1914Deepak", // Your App Password
    },
});

// Define email options
const mailOptions = {
    from: "deepakjgenai@gmail.com", // Sender address
    to: "deepakcolab2024@gmail.com", // Recipient address
    subject: "Booking Confirmation", // Subject line
    text: `Booking Confirmed!

    Museum: Bharathiar Museum
    Tickets: 3
    Total Paid: ₹15.00
    Admin UPI ID: admin@bank
    Your UPI ID: user1@bank

    Thank you for your booking!`, // Plain text body
};

// Send the email
transporter.sendMail(mailOptions, (error, info) => {
    if (error) {
        console.log("Error sending email:", error);
    } else {
        console.log("Email sent:", info.response);
    }
});

File: backend/temp.py
import smtplib
from email.mime.text import MIMEText
from email.mime.multipart import MIMEMultipart

# Your email credentials
sender_email = "deepakjgenai@gmail.com"
receiver_email = "deepakcolab2024@gmail.com"
password = "zvhd brey bfks cfgl"  # Your app password

# Set up the server
server = smtplib.SMTP("smtp.gmail.com", 587)
server.starttls()
server.login(sender_email, password)

# Compose the email
subject = "Success"
body = "This is the message body indicating success of the mail."

msg = MIMEMultipart()
msg["From"] = sender_email
msg["To"] = receiver_email
msg["Subject"] = subject
msg.attach(MIMEText(body, "plain"))

# Send the email
server.sendmail(sender_email, receiver_email, msg.as_string())

# Close the server connection
server.quit()

print("Email sent successfully!")
